### ISSUE TITLE
Subtle typo causes silent failure

### ISSUE THREAD

**Issue by chauncey-garrett** [jekyll/jekyll-redirect-from#70]
I came across this issue by accident. 

In my YAML front matter, I accidentally used `redirect-from:` instead of `redirect_from:` on a couple of posts. Instead of simply ignoring the incorrect form or sending an error to the terminal, `jekyll-redirect-from` silently allows the site to be built but *without* inserting redirects.

**Comment by parkr** [comment]
I could see the plugin saying "I have nothing to do!" but I don't think we'd look for typos. That would get messy really quickly.

**Comment by chauncey-garrett** [comment]
Understandable. I should've mentioned that the typo broke the redirect for all pages—including those that had the correct spelling.

**Comment by parkr** [comment]
> the typo broke the redirect for all pages—including those that had the correct spelling.

Now that seems fishy. The code shouldn't choke on that. Can you create a simple example site that displays this behaviour?

**Comment by chauncey-garrett** [comment]
> Now that seems fishy. The code shouldn't choke on that.

Agreed.

> Can you create a simple example site that displays this behaviour?

I made a test site but couldn't reproduce the error I saw earlier. I'm not sure what it was along the way that led me to believe there was an issue with the plugin. I went back and undid my commit fix to try and reproduce but couldn't. Everything worked as you'd expect: typo = no redirect; no typo = redirect.

The good news is my site works now and your plugin doesn't appear to have any issues! Sorry for the false positive and thanks for the help!
